### ISSUE TITLE
Activity graph: show average steps?

### ISSUE THREAD

**Issue by IzzySoft** [Freeyourgadget/Gadgetbridge#354]
The activity graph for steps currently shows only the target (10k steps) on the upper-right of the graph. Could we also have the average (week/all) here?

**Comment by cpfeiffer** [comment]
Sure, yes. Adding a link to #301 so it gets done there.

**Comment by IzzySoft** [comment]
Good point. Already thought you might not wish to put too much work into the "old GUI" if a new one's in the make. Btw, seeing that first screenshot: the upper-right shows a refresh symbol. I never saw that on my installation. Is that device dependent (as e.g. it's not possible to say "Pebble, gimme updates" for Pebble only gives its updates when it decides to itself)?

**Comment by danielegobbetti** [comment]
Precisely. The pebble activity is sent by the watch when it feels like doing so. There are ways to force the transfer but they're initiated from the watch.
A GB watchapp *could* be instructed by GB on the android device to tell the watch to send data but this is a bit far fetched (and would fall into the watchapp-issue :-) )

**Comment by IzzySoft** [comment]
OK. So with all that postponed to the redesign: Hope you don't mind if I postpone the corresponding wiki page as well? Or does it make much sense to start on that before?

**Comment by cpfeiffer** [comment]
No, take your time, it'll take some time for us as well :-)

**Comment by IzzySoft** [comment]
OK, fine with me! Better to start fresh then. Reworking an existing article often is tougher than writing it from the scratch :)
